src/main/java/edu/ie3/util/quantities/QuantityUtil.java

```diff
@@ -31,6 +36,24 @@ private QuantityUtil() {
     return Quantities.getQuantity(quantity.getValue(), quantity.getUnit());
   }
 
+  /**
+   * Compares two {@link Quantity}s, if they are considerably equal. This is foremost important for
+   * {@link tech.units.indriya.quantity.NumberQuantity}s with internal big decimal or double values.
+   * The comparison is made on the absolute difference of both quantities' value. Both quantities
+   * are converted into a's unit before the comparison. Internally calls {@link
+   * QuantityUtil#considerablyAbsEqual(Quantity, Quantity, double)} with a default tolerance of
+   * {@value DEFAULT_ABSOLUTE_TOLERANCE}
+   *
+   * @param a First quantity to compare
+   * @param b Second quantity to compare
+   * @param <Q> Type of Quantity
+   * @return true, if both quantities' values differ less then the given tolerance else false (based
+   *     on a's unit)
+   */
+  public static <Q extends Quantity<Q>> boolean considerablyAbsEqual(Quantity<Q> a, Quantity<Q> b) {
```

Comment:
As the indriya lib is distinguishing between `equals` and `isEquivalentTo`, I would vote for adapting to this taxonomy. In my mind "is the same" and "equals" colloquial is the same. Therefore, renaming the `considerablyEqual` methods to `considerablyEquivalentTo` and `isTheSame` to `equals` would clarify, what the methods actually do. Having the downside... to be breaking.

@johanneshiry What are your thoughts on this?

Reply:
I would like that! Ideally, we'd find a good naming distinction between `considerablyAbsEqual` and `isTheSameConsideringEmpty` as well.

As 2.0 is breaking anyway, I feel like it wouldn't be that bad?

Reply:
I think, we haven't decided, yet, if we will schedule this for a major or a minor version increase. However, from my point of view, it is better to break something irritating now, than sticking to spreading this.